packages/components/src/feature_alert/index.jsx

```diff
@@ -0,0 +1,124 @@
+import React, { useState, useRef, useCallback } from 'react';
+import PropTypes from 'prop-types';
+import clsx from 'clsx';
+import { invoke } from 'nebenan-helpers/lib/utils';
+import CrossIcon from '@goodhood/icons/lib/16x16/cross_filled';
+import BaseTooltip from '../base_tooltip';
+
+import {
+  useEscHandler,
+  useOutsideClick,
+  useDelayedOpen,
+} from './hooks';
+import { getTriggerProps } from './utils';
+import {
+  POSITION_TOP,
+  POSITION_BOTTOM,
+  POSITION_LEFT,
+  POSITION_RIGHT,
+  TRIGGER_HOVER,
+  TRIGGER_CLICK,
+  TRIGGER_DELAYED,
+} from '../base_tooltip/constants';
+import styles from './index.module.scss';
+
+
+const FeatureAlertTooltip = (props) => {
+  const {
+    position,
+    fallbackPosition,
+    trigger,
+    content,
+    children,
+    closeIcon,
+    defaultOpen,
+    onOpen,
+    onClose,
+    className,
+    ...cleanProps
+  } = props;
+
+  const [isOpen, setOpen] = useState(defaultOpen);
+  const rootRef = useRef(null);
+
+  const wasActiveOnce = useRef(false);
+
+  const handleOpen = useCallback((event) => {
+    if (event) event.stopPropagation();
+    if (isOpen || wasActiveOnce.current) return;
+    wasActiveOnce.current = true;
+    setOpen(true);
+    invoke(onOpen);
+  }, [isOpen, onOpen]);
+
+  const handleClose = useCallback(() => {
+    if (!isOpen) return;
+    setOpen(false);
+    invoke(onClose);
+  }, [isOpen, onClose]);
+
+  useEscHandler(handleClose);
+  useOutsideClick(rootRef, handleClose, closeIcon);
+  useDelayedOpen(trigger, wasActiveOnce, handleOpen);
+
+  const rootClassName = clsx(styles.root, className);
+  const tooltipClassName = clsx(styles.tooltip, { [styles.isActive]: isOpen });
+
+  const triggerProps = getTriggerProps(trigger, handleOpen);
+
+  const bubble = (
+    <aside className={styles.content}>
+      {content}
+      {closeIcon && <span onClick={handleClose} className={styles.cross}><CrossIcon /></span>}
+    </aside>
+  );
+
+  return (
+    <article {...cleanProps} className={rootClassName} ref={rootRef}>
+      <BaseTooltip
+        bubble={bubble} position={position}
+        tooltipClassName={tooltipClassName}
+        arrowClassName={styles.arrow}
+        triggerProps={triggerProps}
+      >
+        <div {...triggerProps}>{children}</div>
+      </BaseTooltip>
+    </article>
+  );
+};
+
+FeatureAlertTooltip.defaultProps = {
+  position: POSITION_LEFT,
+  fallbackPosition: POSITION_RIGHT,
+  closeIcon: false,
+  defaultOpen: false,
+};
+
+FeatureAlertTooltip.propTypes = {
+  className: PropTypes.string,
+  position: PropTypes.oneOf([
+    POSITION_TOP,
+    POSITION_BOTTOM,
+    POSITION_LEFT,
+    POSITION_RIGHT,
+  ]),
+  fallbackPosition: PropTypes.oneOf([
+    POSITION_TOP,
+    POSITION_BOTTOM,
+    POSITION_LEFT,
+    POSITION_RIGHT,
+  ]),
```

Comment:
keep props the same as they previously were - we will add props when we need them